.nuspec/maui.aotprofile.txt

```diff
@@ -3257,12 +3223,6 @@ Methods:
 	System.Reflection.CustomAttributeTypedArgument System.Array:InternalArray__get_Item (int)
 	System.Reflection.CustomAttributeTypedArgument System.Collections.ObjectModel.ReadOnlyCollection`1<System.Reflection.CustomAttributeTypedArgument>:System.Collections.Generic.IList<T>.get_Item (int)
 	System.Reflection.CustomAttributeTypedArgument[] System.Reflection.RuntimeCustomAttributeData:UnboxValues (object[])
-	System.Reflection.Emit.ILGenerator System.Reflection.Emit.DynamicMethod:GetILGenerator ()
-	System.Reflection.Emit.ILGenerator System.Reflection.Emit.DynamicMethod:GetILGenerator (int)
-	System.Reflection.Emit.Label System.Reflection.Emit.ILGenerator:BeginExceptionBlock ()
-	System.Reflection.Emit.Label System.Reflection.Emit.ILGenerator:DefineLabel ()
-	System.Reflection.Emit.LocalBuilder System.Reflection.Emit.ILGenerator:DeclareLocal (System.Type,bool)
-	System.Reflection.Emit.LocalBuilder System.Reflection.Emit.ILGenerator:DeclareLocal (System.Type)
```

Comment:
@PureWeen this is good, we got rid of these from: https://github.com/xamarin/xamarin-android/commit/2b80d7825478f4845dd623bb027c28f5b78a798a

Reply:
I also removed the SplitDrawable so that might have helped a bit on the startup performance https://github.com/dotnet/maui/pull/9896